nucypher/network/status_app/moe.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, moe, ws_port: int, *args, **kwargs):
         template_path = os.path.join(self.TEMPLATE_DIR, 'moe.html')
         with open(template_path, 'r') as file:
             moe_template = file.read()
-            self.dash_app.index_string = Template(moe_template).substitute(ws_port=ws_port)
+            self.dash_app.index_string = Template(moe_template).substitute(ws_port=ws_port, ws_host='0.0.0.0')  # TODO
```

Comment:
Would we want to pass in `ws_host` to `__init__(...)` and then use its value here instead of just using the default, which can be overridden in the cli?

Reply:
or maybe `moe.host` can just be used?

Reply:
:heavy_check_mark: